examples/orderbook/create-bid-with-nextjs/app/utils/bid.ts

```diff
@@ -0,0 +1,75 @@
+import { orderbook } from "@imtbl/sdk";
+import { Web3Provider } from "@ethersproject/providers";
```

Comment:
`@ethersproject/providers` is missing from the dependencies object in the package.json of this example. Could that please be added there?